x/paramfilter/test/governance_params_test.go

```diff
@@ -0,0 +1,549 @@
+package test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/x/paramfilter"
+	"github.com/stretchr/testify/suite"
+
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	testutil "github.com/celestiaorg/celestia-app/test/util"
+	blobtypes "github.com/celestiaorg/celestia-app/x/blob/types"
+	bsmoduletypes "github.com/celestiaorg/celestia-app/x/blobstream/types"
+	minttypes "github.com/celestiaorg/celestia-app/x/mint/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	distributiontypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	params "github.com/cosmos/cosmos-sdk/x/params"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	ibctransfertypes "github.com/cosmos/ibc-go/v6/modules/apps/transfer/types"
+	ibcclienttypes "github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
+	ibcconnectiontypes "github.com/cosmos/ibc-go/v6/modules/core/03-connection/types"
+)
+
+type TestSuite struct {
+	suite.Suite
+
+	app        *app.App
+	ctx        sdk.Context
+	govHandler v1beta1.Handler
+	pph        paramfilter.ParamBlockList
+}
+
+func (suite *TestSuite) SetupTest() {
+	suite.app, _ = testutil.SetupTestAppWithGenesisValSet(app.DefaultConsensusParams())
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{})
+	suite.govHandler = params.NewParamChangeProposalHandler(suite.app.ParamsKeeper)
+	suite.pph = paramfilter.NewParamBlockList([2]string{})
```

Comment:
Does this do anything?

Shouldn't it be:
```go
suite.govHandler = paramfilter.NewParamBlockList(suite.app.BlockedParams()...).GovHandler(suite.app.ParamsKeeper)
```

Reply:
Yes the code I have works as it's based off the code you mentioned, but without a param block list blocking the proposals.

I have now included the suggested changes in this PR, however I initially did not because of [this comment](https://github.com/celestiaorg/celestia-app/pull/2919#discussion_r1421568115) for testing unmodifiable params, where  I needed to test if a param stated to be unmodifiable in params.md is actually unmodifiable before adding it to `App.BlockedParams()`.